backend/index.ts

```diff
@@ -13,9 +14,9 @@ export interface CreateAuthenticatorOptions {
     // sign count
     getSignCount(key: CryptoKey, rpID: string, credentialID: ArrayBuffer): Promise<number>
 
-    incrementSignCount(key: CryptoKey, rpID: string, credentialID: ArrayBuffer): Promise<void>
+    incrementSignCount(key: CryptoKey, rpID: string, credentialID?: ArrayBuffer | null): Promise<void>
 
-    hasCredential(rpID: string, credentialID: ArrayBuffer): Promise<boolean>
+    hasCredential(rpID: string, credentialID?: ArrayBuffer | null): Promise<boolean>
```

Comment:
Why this change?

Reply:
why not

Reply:
Please introduce why you need this change